src/components/Search/components/SearchQueryItemList.tsx

```diff
@@ -0,0 +1,44 @@
+import { useFetchShops } from 'api/search';
+import PreviousButton from 'components/PreviousButton/PreviousButton';
+import { useParams } from 'react-router-dom';
+import styles from 'pages/Search/Search.module.scss';
+import SearchNavigationBar from './SearchNavigationBar';
+import SearchQueryItem from './SearchQueryItem';
+import LoadingView from './LoadingView';
+
+interface Props {
+  address: string,
+  dist: number,
+  placeId: string,
+  placeName: string,
+  score: number,
+  shopId: number,
+  x: string,
+  y: string,
+}
+
+function SearchQueryItemList() {
+  const { searchQuery } = useParams();
+  const { isFetching, data } = useFetchShops(searchQuery as string);
+
+  return (
+    <div>
+      <div className={styles.search}>
+        <SearchNavigationBar>
+          <PreviousButton />
+          <h1 className={styles['search-nav__text']}>{searchQuery}</h1>
+        </SearchNavigationBar>
+        {
+          <div className={styles['search-detail']}>
+            {isFetching ? <LoadingView />
+              // eslint-disable-next-line max-len
+              : data?.data.content.map((shop: Props) => <SearchQueryItem key={shop.shopId} data={shop} />)}
```

Comment:
props를 다른 이름으로 줄 필요가 없을 것 같아요.
```suggestion
              : data?.data.content.map((shop: Props) => <SearchQueryItem key={shop.shopId} shop={shop} />)}
```

Reply:
위에 코멘트랑 같이 수정했습니다~